test/test_jujuversion.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
```

Comment:
This file is not runnable. It needs unittest.main() under the typical if block, or the line may be dropped.